chore(docs): 06/01/26 docs refinements (#2235)
namehash/ensnode: docs/ensnode.io/src/content/docs/docs/integrate/ens-subgraph/index.mdx

:::danger[🚨 The ENS Subgraph is not ENSv2 compatible]
The ENS Subgraph **fundamentally fails as a source of ENS data as soon as ENSv2 launches.**

<LinkCard

>   title="Fix ENS integrations from breaking"

Reviewer: ```suggestion
  title="Keep ENS apps working 🚨"
```

  href="/docs/integrate/why-ensnode/keep-ens-working"

>   description="See the apps that break when ENSv2 launches, and how ENSNode keeps them working."

Reviewer: ```suggestion
  description="See apps currently set to break when ENSv2 launches unless they upgrade to the new ENS Omnigraph API."
```

/>
:::

> ## ENSNode Subgraph compatability

Author: Applied

> ## ENSNode Subgraph compatability

Author: Applied

> ## ENSNode Subgraph compatability

Author: Applied

> ENSNode maintains a verified Subgraph-compatible API for migrating existing integrations, but it is **not** the path forward for ENSv2.

Reviewer: ```suggestion
ENSNode maintains a verified Subgraph-compatible API for migrating existing integrations from The Graph, but it is **not** the path forward for ENSv2.
```
